monix-execution/shared/src/main/scala/monix/execution/misc/Local.scala

```diff
@@ -114,7 +114,7 @@ object Local {
     }
 
   /** Macros implementations for [[bind]] and [[bindClear]]. */
-  class Macros(override val c: whitebox.Context) extends InlineMacros with HygieneUtilMacros {
+  private class Macros(override val c: whitebox.Context) extends InlineMacros with HygieneUtilMacros {
```

Comment:
Is that okay to make that private? I don't even know how scoping works for macros.

Reply:
me neither but it shouldn't compile if it's wrong, right? Or is there a way it could still fail somewhere?

Reply:
That's the point, I have no clue 😆

Reply:
@oleg-py 

I changed `package` of `LocalSuite` to something completely else and it passed
Also moved it to `monix-eval` subproject with different package (without monix prefix) and worked as well

Looks like it can be private as long as it is in scope of public methods accessing it. I asked someone with some experience with macros and that was his thought too